app/src/main/java/com/framgia/soundclound/data/source/remote/TrackRemoteDataSource.java

```diff
@@ -0,0 +1,76 @@
+package com.framgia.soundclound.data.source.remote;
+
+import com.framgia.soundclound.data.model.Collection;
+import com.framgia.soundclound.data.model.CollectionResult;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.data.source.TrackDataSource;
+import com.framgia.soundclound.util.Constant;
+import com.framgia.soundclound.util.StringUtil;
+import com.google.gson.Gson;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
```

Comment:
ERROR: Unused import: java.util.logging.Logger. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports